libkbfs/tlf_handle.go

```diff
@@ -522,20 +522,20 @@ func (h *TlfHandle) GetCanonicalName() CanonicalTlfName {
 	return h.name
 }
 
-func buildCanonicalPath(public bool, canonicalName CanonicalTlfName) string {
+// BuildCanonicalPath returns full path for tlf name
+func BuildCanonicalPath(public bool, tlfName CanonicalTlfName) string {
 	var folderType string
 	if public {
 		folderType = "public"
 	} else {
 		folderType = "private"
 	}
-	// TODO: Handle windows paths?
-	return fmt.Sprintf("/keybase/%s/%s", folderType, canonicalName)
+	return fmt.Sprintf("/keybase/%s/%s", folderType, tlfName)
```

Comment:
I think we want this /keybase/private/username string as the canonical universal path that is representative of a path on a platform... we would convert it to windows path on windows? Otherwise it's not canonical.

`/keybase/private/username` - Canonical path

`k:\private\username` - Windows (if k: configured, could be another drive letter)
`/keybase/private/username` - Linux and darwin (happens to be same as canonical)
`sftp://keybase.priv/username` - Some theoretical other thing